src/Illuminate/Auth/EloquentUserProvider.php

```diff
@@ -155,6 +155,24 @@ public function validateCredentials(UserContract $user, array $credentials)
         return $this->hasher->check($plain, $user->getAuthPassword());
     }
 
+    /**
+     * Rehash the user's password if required and supported.
+     *
+     * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
+     * @param  array  $credentials
+     * @return string|null
```

Comment:
need to recheck the return type docblocks on the three concrete versions of this method, the contract one is correct.

Reply:
Nice catch! Fixed. 👍